### DISCUSSION TITLE
Do you know of ports for other board games or with user input for text on dividers?

### DISCUSSION THREAD

**Discussion by jho1965us** [sumpfork/dominiontabs#549] [General]
Or other alternative ways for generating dividers with text may be template document where text can be entered manual.

In particular I am looking in to doing some tabs for Sagrada, there will probably be less than 10 dividers so I do not mind if I have to enter the text myself.

**Comment by sumpfork** [comment]
I don't offhand, but I have thought about making a generic version of the dominion divider generator. A lot of the draw code and everything would apply, it would just take stripping out all of the special database and image handling.

But, I have too many side projects going as is, so this is not coming from me anytime soon :).
